src/main/java/seedu/address/model/applicant/Applicant.java

```diff
@@ -45,6 +46,7 @@ public Applicant(Name name, Phone phone, Email email, Address address, Set<Tag>
         this.interviewDate = interviewDate;
         this.job = job;
         this.qualification = qualification;
+        this.applicantStatus = new ApplicantStatus(2);
```

Comment:
Since this is a constructor for the Edit method, the parameter in the constructor should take in ApplicantStatus and line 49 should change to this.applicantStatus = applicantStatus.